### ISSUE TITLE
Update and fixes

### ISSUE THREAD

**Issue by Lancelotbronner** [datadever/IncrementalIndustries#8]
On my personal modification of the pack, I've updated every mods to their latest versions and fixed up the KubeJS script.

I've also removed FabricTreeChopper as it has some misinteractions with ultimine and WhereIsIt because it crashes with latest mod versions.

If you'd like I could make a pull requests, so far I'm really enjoying this pack!

**Comment by Lancelotbronner** [comment]
The updates do fix a bunch of couple issues and slowdowns I had experienced and as far as I'm aware it doesn't affect balance

**Comment by Annnn-n-n-n-nthon** [comment]
@Lancelotbronner I'm new to modded and am having an issue where my game crashes whenever I open a Create ponder tutorial, did you ever run into this issue? And did your updates and edits to KubeJS fix it?

**Comment by Lancelotbronner** [comment]
I don't remember, I haven't been able to play for a year at least!

I don't now if the pack has been updated since I played it, you can try my edits as they did give me a working version for a server with friends but by now even that is probably outdated.

**Comment by Annnn-n-n-n-nthon** [comment]
Yeah, unfortunately it seems to be a outdated project...I think I'll be better off just finding a newer pack with Create in it. Thank you though!

**Comment by JudahShmoo** [comment]
How did you fix the KubeJS script? I am having trouble with it not working after having updated the mods.
When I open a world, I get this in the chat:
```
KubeJS errors found [2]! Run '/kubejs errors' for more info
[1] Error occurred while handling event 'item.tags': Assignment to undeclared variable lv_cables (server_scripts:script.js#1989)
[2] Error occurred while handling event 'recipes': Assignment to undeclared variable gold_tier_kibes (server_scripts:script.js#808)
More info in 'logs/kubejs/server.txt'
2 warnings found. Run '/kubejs warnings' to see them
[1] Failed to parse recipe 'wands:netherite_wand[minecraft:smithing_transform]'! Falling back to vanilla: java.lang.NullPointerException: Unknown recipe type: minecraft:smithing_transform
[2] Failed to parse recipe wands:netherite_wand[minecraft:smithing_transform]: com.google.gson.JsonSyntaxException: Invalid or unsupported recipe type 'minecraft:smithing_transform'
```
I also noticed I can't get the Mob Slaughtering Drill and Brass dust doesn't smelt.

**Comment by Lancelotbronner** [comment]
Oh man it has been years.. I just checked and I don't have any traces of this pack anywhere on my computer, sorry.
